causalml/inference/tree/plot.py

```diff
@@ -0,0 +1,168 @@
+"""
+Visualization functions for forest of trees-based ensemble methods for Uplift modeling on Classification
+Problem. 
+"""
+
+from __future__ import print_function
+from collections import defaultdict
+import numpy as np
+import pandas as pd
+import pydotplus
+
+def uplift_tree_string(decisionTree, x_names):
```

Comment:
@zhenyuz0500 Why don't we make `uplift_tree_string()` and `uplift_tree_plot()` as member functions of `DecisionTree`, and add `plot()` to `UpliftTreeClassifier` to call `fitted_uplift_tree.plot()` within.

So, at the end, users can simply call `uplift_model.plot()` to visualize the uplift tree.